query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryFunctionConstants.java

```diff
@@ -0,0 +1,15 @@
+package org.hypertrace.core.query.service;
+
+import org.hypertrace.core.query.service.api.Expression;
+import org.hypertrace.core.query.service.api.Function;
+
+/** A converter from the canonical format received in a query to the appropriate dat */
```

Comment:
typo: "dat"?

Reply:
It looks like I got distracted in the midst of writing this comment... And then I changed this from an implemented interface to constants.

```suggestion
/**
 * Canonical query function names received in requests. These should be converted to data-store
 * specific names when handling a request.
 */
```